plugins/jevents/jevents.php

```diff
@@ -65,7 +65,7 @@ function onContentBeforeSave($context, $data)
 			if ($result_count >= 1)
 			{
 				JFactory::getApplication()->enqueueMessage(JText::sprintf('JEV_CAT_MAN_DELETE_WITH_IDS',  $result_count), 'Warning');
-				JFactory::getApplication()->enqueueMessage(JText::_('JEV_CAT_DELETE_MSG_EVENTS_FIRST'), 'Warning');
+				JFactory::getApplication()->enqueueMessage(JText::sprintf('JEV_CAT_DELETE_MSG_EVENTS_FIRST'), 'Warning');
```

Comment:
I made the change as I was planning on extending it to pass through a variable later, so saw no harm in adding it now.